src/Databases/DatabaseOnDisk.cpp

```diff
@@ -119,6 +119,8 @@ String getObjectDefinitionFromCreateQuery(const ASTPtr & query)
         throw Exception(ErrorCodes::LOGICAL_ERROR, "Query '{}' is not CREATE query", query_buf.str());
     }
 
+    addColumnsDescriptionToCreateQueryIfNecessary(*create, columns_description);
```

Comment:
I don't think that it's suitable place for adding columns definition to AST, it would be better to do it somewhere in `InterpreterCreateQuery::getTablePropertiesAndNormalizeCreateQuery`

Reply:
I couldn't add it to `InterpreterCreateQuery::getTablePropertiesAndNormalizeCreateQuery` because table creation happens after calling this function, but it was a good idea to move it in `InterpreterCreateQuery`, I placed adding columns to query just after table creation.